python_modules/dagster/dagster_tests/execution_tests/test_logical_versions.py

```diff
@@ -29,19 +29,30 @@
 )
 from dagster._core.definitions.observe import observe
 from dagster._core.execution.execute_in_process_result import ExecuteInProcessResult
+from dagster._core.instance_for_test import instance_for_test
 from typing_extensions import Literal
 
 # ########################
 # ##### HELPERS
 # ########################
 
 
+class MaterializationTable:
+    def __init__(self, materializations: Mapping[AssetKey, AssetMaterialization]):
+        self.materializations = materializations
+
+    def __getitem__(self, key: Union[str, AssetKey]) -> AssetMaterialization:
```

Comment:
do we really need to overload an operator here? Seems unnecessarily clever/not amenable for grepping with limited upside

Reply:
It's just a convenience used in one test file, returned by a few helpers in that file, to avoid cluttering tests with `AssetKey(...)`. Necessary? No. Nicer to use when writing tests? IMO yes, but just a matter of taste. I don't have an aversion to using `__getitem__` for this kind of thing but I'm happy to change it.